ksql-engine/src/test/java/io/confluent/ksql/util/KafkaTopicClientTest.java

```diff
@@ -316,12 +317,16 @@ private DescribeClusterResult getDescribeClusterResult() {
     return describeClusterResult;
   }
 
-  private DescribeConfigsResult getDescribeConfigsResult() {
+  private DescribeConfigsResult getDescribeConfigsResult(boolean isCompacted) {
     DescribeConfigsResult describeConfigsResult = mock(DescribeConfigsResult.class);
-    ConfigEntry configEntry = new ConfigEntry("delete.topic.enable", "true");
+    ConfigEntry configEntryDeleteEnable = new ConfigEntry("delete.topic.enable", "true");
+    ConfigEntry configEntryIsCompacted = new ConfigEntry("cleanup.policy",
```

Comment:
Is this returned config entry actually used anywhere in admin client? The only usage of describeConfigs I see is to get the broker's delete.topic.enable config. If cleanup.policy is not read anywhere can we just get rid of it and the isCompacted parameter to this function.

Reply:
No it was only used in tests. Removed it.